src/service/run-api-service.js

```diff
@@ -494,6 +495,11 @@ module.exports = function (config) {
                 runService: this
             }));
             return vs;
+        },
+
+        introspection: function (config) {
```

Comment:
Is there a reason on why expose the service from the RunAPI service? We mostly use the higher level RunManager when working with runs. I suggest exposing it in the [app.js](https://github.com/forio/epicenter-js-libs/blob/master/src/app.js) file and/or the run-manager